src/pages/admin/entities/[entityId].svelte

```diff
@@ -36,11 +36,11 @@ const onSave = async () => {
     <div class="my-1">
       <h4>Code: {entity.code}</h4>
 
-      <TextFieldWithLabel label="Name" bind:value={entity.name} />
+      <TextField label="Name" bind:value={entity.name} />
 
-      <TextFieldWithLabel label="Income Account" bind:value={entity.income_account} />
+      <TextField label="Income Account" bind:value={entity.income_account} />
 
-      <TextFieldWithLabel label="Parent Entity" bind:value={entity.parent_entity} />
+      <TextField label="Parent Entity" bind:value={entity.parent_entity} />
```

Comment:
Assuming this is actually required
```suggestion
      <TextField required label="Parent Entity" bind:value={entity.parent_entity} />
```